vocabs/oada/index.js

```diff
@@ -289,6 +289,20 @@ register('time', override('generic-data-value', {
   },
 }));
 
+register('led-status', {
+  description: 'Status of LED light. Typically used for debugging applications ' +
+      'Could be used to represent on/off status or red/green status, etc',
+  type: Boolean,
+});
+
+register('network-strength', override('generic-data-value', {
+  description: 'Strength of wireless network connection in -dBm ' +
+      'ranging from ~-10dBm to ~-100dBm',
+  properties: {
+    'units': enumSchema( [ 'dBm' ] ),
+  },
+}));
```

Comment:
network-strength is good here, but you didn't use it up above in the heartbeat?  you used cell_ns instead?

Reply:
I see, you overrode network-strength below for cell_ns.  Your example above should have made cell_ns an object then, because the vocab says it should be an object with keys value and units (units go in the template, value goes in the data points)